test/integrations/destinations/adobe_analytics/dataDelivery/business.ts

```diff
@@ -0,0 +1,181 @@
+import { ProxyMetdata } from '../../../../../src/types';
+import { ProxyV1TestData } from '../../../testTypes';
+import { generateProxyV1Payload } from '../../../testUtils';
+
+const statTags = {
+  aborted: {
+    destType: 'ADOBE_ANALYTICS',
+    destinationId: 'dummyDestinationId',
+    errorCategory: 'dataValidation',
+    errorType: 'instrumentation',
+    feature: 'dataDelivery',
+    implementation: 'native',
+    module: 'destination',
+    workspaceId: 'dummyWorkspaceId',
+  },
+};
+
+export const proxyMetdata: ProxyMetdata = {
+  jobId: 1,
+  attemptNum: 1,
+  userId: 'dummyUserId',
+  sourceId: 'dummySourceId',
+  destinationId: 'dummyDestinationId',
+  workspaceId: 'dummyWorkspaceId',
+  secret: {},
+  dontBatch: false,
+};
+const headers = {
+  'Content-Type': 'application/xml',
+};
+
+export const reqMetadataArray = [proxyMetdata];
+
+const failureRequestParameters = {
+  XML: {
+    payload:
+      '<?xml version="1.0" encoding="utf-8"?><request><browserHeight>1794</browserHeight><browserWidth>1080</browserWidth><campaign>sales campaign</campaign><channel>web</channel><currencyCode>USD</currencyCode><ipaddress>127.0.0.1</ipaddress><language>en-US</language><userAgent>Dalvik/2.1.0 (Linux; U; Android 9; Android SDK built for x86 Build/PSR1.180720.075)</userAgent><referrer>https://www.google.com/search?q=estore+bestseller</referrer><marketingcloudorgid/><events>prodView</events><products>Games;;11;148.39</products><reportSuiteID>failureReport</reportSuiteID></request>',
```

Comment:
this repeated

Reply:
![image](https://github.com/rudderlabs/rudder-transformer/assets/33080863/2d00295e-715d-4e7a-bc2e-12d72035978a)

Reply:
WIll handle in the upcoming network handler card